### PR TITLE
Create CODE_OF_CONDUCT.md

### LINKED ISSUE #44: Chore: add code of conduct file
### Type of feature

📦 Chore

### Current behavior

This repo does not have a code of conduct file

### Suggested solution

Create a new file named `CODE_OF_CONDUCT.md` and use the content of https://www.contributor-covenant.org/version/2/1/code_of_conduct/

### Additional context

_No response_

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by Madhvik007**
## What type of PR is this? (check all applicable)

- [ ] 💡 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] ⏩ Revert

## Description
This PR adds CODE_OF_CONDUCT.md

## Related Tickets & Documents
Fixes #44

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (takanome-dev) [inline CODE_OF_CONDUCT.md:63]:
```suggestion
reported by contacting the project owner at [takanomedev@duck.com](mailto:takanomedev@duck.com)
```

**Review by takanome-dev — CHANGES_REQUESTED** [review]
Thanks for opening this pr, @Madhvik007. I updated the pr template 👌
Just a couple of changes and this should be good to go :rocket: 

Please add the attribution section at the end of the file

**Comment by Madhvik007** [comment]
I've made the changes. Is there anything else you'd like me to do?

**Review by takanome-dev — APPROVED** [review]
LGTM

**Comment by takanome-dev** [comment]
> I've made the changes. Is there anything else you'd like me to do?

No, everything looks good. Thanks again :+1:

**Pull request merged**